localstack/services/secretsmanager/provider.py

```diff
@@ -515,6 +514,22 @@ def fake_secret_remove_version_stages_from_old_versions(fn, self, version_stages
         del self.versions[version_no_stages]
 
 
+@patch(SecretsManagerResponse.rotate_secret, pass_target=False)
+def rotate_secret(self) -> str:
```

Comment:
I'd suggest adding a comment as to why we patch this. Otherwise this knowledge gets lost pretty quickly and we can't easily reassess if we still need that patch